MetaMask/metamask-extension
# Enable Linea

## **Description**

This PR enables Linea in Swaps.

TODO
- [x]  update MetaSwap Contract address for Linea once deployed

## Manual Testing Steps
1. Run the extension locally via this command: `SWAPS_USE_DEV_APIS=1 yarn start`
1. Check Swaps
